corehq/apps/accounting/async_handlers.py

```diff
@@ -161,14 +161,15 @@ def country_response(self):
         from django_countries.data import COUNTRIES
         countries = sorted(COUNTRIES.items(), key=lambda x: x[1].encode('utf-8'))
         if self.search_string:
-            return filter(lambda x: x[1].lower().startswith(self.search_string.lower()), countries)
+            search_string = self.search_string.capitalize()
```

Comment:
All the countries are Capitalised. So thought would be better to just capitalise the search string instead of doing calling lower on both.
How is this brittle?